example/urls.py

```diff
@@ -16,7 +17,8 @@
     url(r'^send-email/$', views.send_email, name='send-email'),
     url(r'^reporting/', include('django_ses.urls')),
 
-    url(r'^event/', 'django_ses.views.handle_event', name='handle_event'),
+    url(r'^bounce/', 'django_ses.views.handle_bounce', name='handle_bounce'),       # Deprecated, see SESEventWebhookView.
+    url(r'^event-webhook/', SESEventWebhookView.as_view(), name='django_ses_event_webhook'),
```

Comment:
I don't think you need "django_ses" in the name. The package has its own prefix. Perhaps just "event_webhook"